src/components/drop-down/drop-down.st.css

```diff
@@ -0,0 +1,27 @@
+.root {
+    width: 276px;
+    height: 36px;
+    font-family: Arial, sans-serif;
+    font-size: 16px;
+}
+
+.drop-down-label {
+    border: 1px solid #adbac5;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    .caret {
```

Comment:
might be a confusing name but its actually for the whole `<DropDownLabel />` component that contains the label string and and the `caret`